src/main/java/minicraft/saveload/Save.java

```diff
@@ -159,6 +165,26 @@ public void writeToFile(String filename, List<String> savedata) {
 		
 		Renderer.render(); // AH HA!!! HERE'S AN IMPORTANT STATEMENT!!!!
 	}
+	public void writeToFile(String filename, byte[] savedata) {
+		try {
+			FileOutputStream out = new FileOutputStream(filename);
+			DeflaterOutputStream dos = new DeflaterOutputStream(out, new Deflater(9));
+			dos.write(savedata);
+			dos.flush();
+			dos.close();
+		} catch (IOException ex) {
+			ex.printStackTrace();
+		}
+		
+		data.clear();
+		
+		LoadingDisplay.progress(7);
+		if(LoadingDisplay.getPercentage() > 100) {
```

Comment:
Follow the formatting norms please

Reply:
No, the formatting of the code. You need to follow the already established norms for how the code will look. Take a look at how other files are formatted and copy that.